README.md

```diff
@@ -90,3 +90,9 @@ In Kratos Core:
 In applications
 - [Trilinos](https://trilinos.org/) for MPI linear algebra and solvers used in trilinos application
 - [METIS](http://glaros.dtc.umn.edu/gkhome/views/metis) for partitioning in metis application
+
+# How to cite Kratos?
+Please, use the following references when citing Kratos in your work.
+- Dadvand, P., Rossi, R. & Oñate, E. An Object-oriented Environment for Developing Finite Element Codes for Multi-disciplinary Applications. Arch Computat Methods Eng 17, 253–297 (2010). https://doi.org/10.1007/s11831-010-9045-2
+- Dadvand, P., Rossi, R., Gil, M., Martorell, X., Cotela, J., Juanpere, E., Idelsohn, S., Oñate, E. (2013). Migration of a generic multi-physics framework to HPC environments. Computers & Fluids. 80. 301–309. 10.1016/j.compfluid.2012.02.004.
+- Mataix Ferrándiz, V., Bucher, P., Rossi, R., Cotela, J., Carbonell, J. M., Zorrilla, R., … Tosi, R. (2020, May 29). KratosMultiphysics (Version 8.0). Zenodo. https://doi.org/10.5281/zenodo.3234644
```

Comment:
this is determined by the number of commits right?

Just clarifying such that it is fair for everyone

Reply:
I guess so, but I cannot confirm. @roigcarlo created the release in zenodo and maybe he knows more.

Reply:
I agree :) ...

Reply:
This list is automatically generated from  the public info available in github and as you say is ordered by the number of commits. This ordering is not enforced and I can change it any time you want. I can also add a ORCID to your profile etc...  / Change affiliation center(s) / change your name etc...

If you have any request please tell me. Also changing that will not change the DOI number whatsoever, so please don't wait for a new release. (send me the requests to croig@cimne.upc.edu)

Also worth mention that the DOI citation for the Kratos repo references only the last release of that software in zenodo page, and no other derived works /versions.